packages/element/src/platform.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Parts of this source were derived and modified from react-native-web,
+ * released under the MIT license.
+ *
+ * Copyright (c) 2016-present, Nicolas Gallagher.
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ */
+const Platform = {
+	OS: 'web',
+	select: ( spec ) => ( 'web' in spec ? spec.web : spec.default ),
+};
+/**
+ * Component used to detect the current Platform being used.
+ * Use Platform.OS === 'web' to detect if running on web enviroment.
+ *
+ * This is the same concept as the React Native implementation.
+ *
+ * @see https://facebook.github.io/react-native/docs/platform-specific-code#platform-module
+ *
+ * Here is an example of how to use the select method:
+ * @example
+ * ```js
+ * const placeholderLabel = Platform.select( {
```

Comment:
One more nit before you merge:
```suggestion
import { Platform } from '@wordpress/element'; 

 * const placeholderLabel = Platform.select( {
```
to make it easier to copy for folks.